### PR TITLE
F #src 3672 left teleop

### PR DESCRIPTION
**Pull request by ethanfowler**
## Proposed changes

Please explain the changes you made. Add pictures or videos to explain them better if appropiate.

## Types of changes

What types of changes does your code introduce?

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

Put an x in the boxes that apply. You can also fill these out after creating the PR. This is a reminder of what we should look for before merging this code. I have:

- [x] Read and follow the [contributing guidelines](https://github.com/shadow-robot/sr_documentation/blob/master/CONTRIBUTING.md).
- [x] Checked that all tests pass with my changes
- [ ] Added tests (automatic or manual) that prove the fix is effective or that the feature works
- [ ] Added necessary documentation (if appropriate)
- [x] Added the corresponding license to each file and add the current year of any one that you modified.
- [x] Tested on real hardware (if appropriate)
￼

### CODE REVIEW

File: sr_multi_moveit/sr_box_ur10_moveit_config/config/robot_configs/right_ur10.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ robot:
   name: ursrbox
   manipulators:
     right_manipulator:
-      side: right
+      side: left
```
Comment (beatrizleon) [inline sr_multi_moveit/sr_box_ur10_moveit_config/config/robot_configs/right_ur10.yaml]: left? this is right_ur10.yaml

File: sr_multi_moveit/sr_box_ur10_moveit_config/launch/moveit.rviz [hunk 2]
```diff
@@ -1,20 +1,23 @@
 Panels:
   - Class: rviz/Displays
-    Help Height: 84
+    Help Height: 0
```
Comment (beatrizleon) [inline sr_multi_moveit/sr_box_ur10_moveit_config/launch/moveit.rviz]: Can we revert this to kinetic-devel moveit.rviz?

File: sr_multi_moveit/sr_multi_moveit_config/launch/moveit.rviz [hunk 3]
```diff
@@ -1,23 +1,26 @@
 Panels:
```
Comment (beatrizleon) [inline sr_multi_moveit/sr_multi_moveit_config/launch/moveit.rviz]: Can we revert this to kinetic-devel moveit.rviz?

File: sr_robot_launch/config/gazebo/controller/la_group_position_controller.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+la_group_position_controller:
+  type: "position_controllers/JointGroupPositionController"
+  joints:
+    - la_shoulder_pan_joint
+    - la_shoulder_lift_joint
+    - la_elbow_joint
+    - la_wrist_1_joint
+    - la_wrist_2_joint
+    - la_wrist_3_joint
+    - rh_WRJ1
```
Comment (beatrizleon) [inline sr_robot_launch/config/gazebo/controller/la_group_position_controller.yaml]: should be lh for the wrist

File: sr_robot_launch/launch/sr_ur10arm_box.launch [hunk 5]
```diff
@@ -11,7 +11,7 @@
   <arg name="planning" default="ompl"/>
   <!-- Set to false if you want to start without guis -->
   <arg name="gui" default="true"/>
-  <arg name="verbose" default="true"/>
+  <arg name="verbose" default="true" />
```
Comment (beatrizleon) [inline sr_robot_launch/launch/sr_ur10arm_box.launch]:
```suggestion
  <arg name="verbose" default="true"/>
```

File: sr_robot_launch/launch/sr_ur10arm_box.launch [hunk 6]
```diff
@@ -21,36 +21,41 @@
...
   <!-- ARM CONFIG-->
-  <arg name="arm_id" default="ra"/>
-  <arg name="side" default="right"/>
-  <arg name="arm_model" default="ur10"/>
-  <arg name="initial_z" default="0.7751" if="$(arg scene)"/>
+  <arg name="arm_id" default="$(arg side_letter)a"/>
+  <arg name="initial_z" default="0.7551" if="$(arg scene)"/>
   <arg name="initial_z" default="0.1" unless="$(arg scene)"/>
   <arg name="arm_robot_hw" default="$(find sr_robot_launch)/config/$(arg side)_ur_arm_robot_hw.yaml"/>
-  <arg name="arm_controller" default="$(find sr_robot_launch)/config/$(arg side)_ur_arm_controller.yaml"/>
```
Comment (beatrizleon) [inline sr_robot_launch/launch/sr_ur10arm_box.launch:43]: Why is this not needed anymore?

**Review by beatrizleon — CHANGES_REQUESTED** [review]

File: sr_multi_moveit/sr_box_ur10_moveit_config/config/robot_configs/left_ur3.yaml [hunk 7]
```diff
@@ -0,0 +1,15 @@
+robot:
+  name: ursrbox
+  manipulators:
+    right_manipulator:
+      side: right
```
Comment (beatrizleon) [inline sr_multi_moveit/sr_box_ur10_moveit_config/config/robot_configs/left_ur3.yaml]:
```suggestion
      side: left
```

File: sr_multi_moveit/sr_box_ur10_moveit_config/config/robot_configs/right_ur10.yaml (hunk 1 above)
Comment (ethanfowler) [reply sr_multi_moveit/sr_box_ur10_moveit_config/config/robot_configs/right_ur10.yaml]: Fixed in https://github.com/shadow-robot/sr_interface/pull/419/commits/69bc13be8e64f769c2179bda3d2bfc388b4bc071

File: sr_multi_moveit/sr_box_ur10_moveit_config/launch/moveit.rviz (hunk 2 above)
Comment (ethanfowler) [reply sr_multi_moveit/sr_box_ur10_moveit_config/launch/moveit.rviz]: Fixed in https://github.com/shadow-robot/sr_interface/pull/419/commits/69bc13be8e64f769c2179bda3d2bfc388b4bc071

File: sr_multi_moveit/sr_multi_moveit_config/launch/moveit.rviz (hunk 3 above)
Comment (ethanfowler) [reply sr_multi_moveit/sr_multi_moveit_config/launch/moveit.rviz]: Fixed in https://github.com/shadow-robot/sr_interface/pull/419/commits/69bc13be8e64f769c2179bda3d2bfc388b4bc071

File: sr_robot_launch/config/gazebo/controller/la_group_position_controller.yaml (hunk 4 above)
Comment (ethanfowler) [reply sr_robot_launch/config/gazebo/controller/la_group_position_controller.yaml]: Fixed in https://github.com/shadow-robot/sr_interface/pull/419/commits/69bc13be8e64f769c2179bda3d2bfc388b4bc071

File: sr_robot_launch/launch/sr_ur10arm_box.launch (hunk 5 above)
Comment (ethanfowler) [reply sr_robot_launch/launch/sr_ur10arm_box.launch]: Fixed in https://github.com/shadow-robot/sr_interface/pull/419/commits/69bc13be8e64f769c2179bda3d2bfc388b4bc071

File: sr_robot_launch/launch/sr_ur10arm_box.launch (hunk 6 above)
Comment (ethanfowler) [reply sr_robot_launch/launch/sr_ur10arm_box.launch:43]: It seems it was never used in the first place, no other mention of `arm_controller` in the file.

File: sr_multi_moveit/sr_box_ur10_moveit_config/config/robot_configs/left_ur3.yaml (hunk 7 above)
Comment (ethanfowler) [reply sr_multi_moveit/sr_box_ur10_moveit_config/config/robot_configs/left_ur3.yaml]: Fixed in https://github.com/shadow-robot/sr_interface/pull/419/commits/69bc13be8e64f769c2179bda3d2bfc388b4bc071

**Review by beatrizleon — APPROVED** [review]

**Pull request merged**